docs/01-start.md

```diff
@@ -653,7 +653,7 @@ AXLearn comes with tooling for provisioning and launching training on public clo
 ### Pre-requisites
 
 We assume you have:
-1. A Google Cloud Platform (GCP) project with TPU quota.
+1. A Google Cloud Platform (GCP) project with TPU quota. To set up a brand new GCP project, please follow [these instructions](05-gcp-setup.md).
```

Comment:
Maybe we can directly link to the script here. The script itself should have instructions on how it should be run at the top of the file, mitigating possible out-of-sync issues. WDYT?

Reply:
Makes sense, then we wouldn't need the "05-gcp-setup.md" file

Reply:
Yeah, seems like we can point users directly to the script.

Reply:
made edits in the latest commit